.changeset/serious-houses-watch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/theme': patch
+---
+
+**DEPRECATED**: The `bursts` object from `BackstagePaletteAdditions` has been depreciated and will be removed in a future release
```

Comment:
I think in addition we can mention that `genPageTheme` now has a third argument that allows you to set the font color, but it defaults to white.

Reply:
I've added a note about this to the changeset 👍